src/components/Carousel/index.tsx

```diff
@@ -20,7 +24,11 @@ export const Carousel = ({
   controlSize = 40,
   loop = true,
   withIndicators = true,
+  isAutoplay = false,
+  delay = 2000,
```

Comment:
もうちょっとdelay遅くした方がいいと思います。
```suggestion
  delay = 4000,
```